p2p/src/secret_connection.rs

```diff
@@ -0,0 +1,418 @@
+//! `SecretConnection`: Transport layer encryption for Tendermint P2P connections.
+
+mod amino_types;
```

Comment:
For me it's important to have some structured for the header of the rust files. Usually I apply the following order: 

* imports
  * std
  * third-party
  * first-party
  * crate
  * super
  * mod
* mods
  * declarations
  * exports